### DISCUSSION TITLE
Missing ElementType.ANNOTATION_TYPE in Tracings annotations

### DISCUSSION THREAD

**Discussion by ajtuss** [micronaut-projects/micronaut-core#6207] [General] [answered]
Hey,

I would like to create an annotation to group a few others, but just realized that none of the tracings annotations hasn’t `ElementType.ANNOTATION_TYPE` (eg. `@SpanTag`, `@ContinueSpan`...)
Is this intentional, or can I do a PR to add it?

Here is an example what I want to achive:
```java
@Retention(RetentionPolicy.RUNTIME)
@Inherited
@Target(value = { ElementType.PARAMETER })
@SpanTag("foo") //non compile here
@PathVariable("foo")
public @interface Foo {
}
```

Regards.

**Comment by graemerocher** [comment, marked as answer]
Sounds reasonable, PRs welcome!
